src/components/organisms/BannerAdmin/BannerAdmin.tsx

```diff
@@ -2,14 +2,15 @@ import React, { useState, useCallback, useRef } from "react";
 
 import { makeUpdateBanner } from "api/bannerAdmin";
 
-import { currentVenueSelector } from "utils/selectors";
+import { AnyVenue } from "types/venues";
 
-import { useSelector } from "hooks/useSelector";
+interface BannerAdminProps {
+  venueId?: string;
+  venue: AnyVenue;
+}
 
-export const BannerAdmin: React.FC = () => {
-  const currentVenue = useSelector(currentVenueSelector);
-  const venueId = currentVenue?.id;
-  const existingBannerMessage = currentVenue?.bannerMessage ?? "";
+export const BannerAdmin: React.FC<BannerAdminProps> = ({ venueId, venue }) => {
```

Comment:
We can likely genericise this so that it can be used for both `BannerAdmin` and `src/components/molecules/AdminVideo/AdminVideo.tsx` as they are basically the same thing.

This would probably involve:

- removing the `venue`/`venueId` props
- adding a `label` prop that would be used in place of the current "Banner Message" text in the `<label>`
- adding a `defaultValue` prop that would be used in place of the current `existingBannerMessage`
- adding an `onSubmit` prop that receives the input field data (`string`)
- genericizing the "Clear Banner"/"Save Banner" labels to something like "Clear"/"Save" or similar (potentially as props with defaults set if we want to make this more customizable)